docs/docs/guides/deployment/hydra-oauth2/hydra-oauth2.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Use Ory Hydra as OAuth2 provider
```

Comment:
Should we say OIDC (OpenID Connect) instead of Oauth2?

Reply:
Sure